mpt/impl/imap-mailbox/external-james/src/test/java/org/apache/james/mpt/imapmailbox/external/james/DeploymentValidationJamesModule.java

```diff
@@ -19,29 +19,38 @@
 
 package org.apache.james.mpt.imapmailbox.external.james;
 
+import org.apache.james.mpt.api.DomainAdder;
 import org.apache.james.mpt.api.HostSystem;
 import org.apache.james.mpt.api.ImapHostSystem;
 import org.apache.james.mpt.api.UserAdder;
 import org.apache.james.mpt.host.ExternalHostSystem;
-import org.apache.james.mpt.imapmailbox.external.james.host.ExternalJamesConfiguration;
-import org.apache.james.mpt.imapmailbox.external.james.host.ExternalJamesConfigurationEnvironnementVariables;
-import org.apache.james.mpt.imapmailbox.external.james.host.ExternalJamesImapHostSystem;
-import org.apache.james.mpt.imapmailbox.external.james.host.ExternalJamesSmtpHostSystem;
-import org.apache.james.mpt.imapmailbox.external.james.host.ExternalJamesUserAdder;
+import org.apache.james.mpt.imapmailbox.external.james.host.DomainsAndUsersAdder;
 import org.apache.james.mpt.imapmailbox.external.james.host.SmtpHostSystem;
+import org.apache.james.mpt.imapmailbox.external.james.host.external.ExternalJamesConfiguration;
+import org.apache.james.mpt.imapmailbox.external.james.host.external.ExternalJamesImapHostSystem;
+import org.apache.james.mpt.imapmailbox.external.james.host.external.ExternalJamesSmtpHostSystem;
 
 import com.google.inject.AbstractModule;
 
-public class ExternalJamesModule extends AbstractModule {
+public class DeploymentValidationJamesModule extends AbstractModule {
```

Comment:
I don't agree about this naming change. What we describe here is a "remote" or "external" James server. It's in no way related to a single `DeploymentValidation` use case